Makefile

```diff
@@ -73,12 +73,26 @@ vet:
 	done
 	@echo -----------------
 
+HAS_GLIDE := $(shell command -v glide)
```

Comment:
That is a good question. Those were in a separate PR already merged onto master. I'll see if a rebase on this branch fixes it.